src/hpc/thread.c

```diff
@@ -449,17 +442,6 @@ int JoinThread(int id)
     return 1;
 }
 
-Int ThreadID(Obj thread)
-{
-    return *(UInt *)(ADDR_OBJ(thread) + 1);
-}
-
-void * ThreadTLS(Obj thread)
-{
-    return *(ThreadLocalStorage **)(ADDR_OBJ(thread) + 1);
```

Comment:
Note the above line was incorrect: The +1 should have been a +0; but since nothing calls this function, it went unnoticed.